ci/docker/tizen/Dockerfile

```diff
@@ -13,3 +17,33 @@ RUN git clone --depth=1 https://chromium.googlesource.com/chromium/tools/depot_t
 # Add engine building tools.
 ADD tools/* /engine/tools/
 ENV PATH=$PATH:/engine/tools
+
+
+#
+# Stage for build-engine-with-efl
+#
+FROM build-engine-base AS build-engine-with-efl
+
+# Install dependencies for building EFL.
+RUN apt-get install -y build-essential check meson ninja-build && \
+    apt-get clean
+RUN apt-get install -y libssl-dev libsystemd-dev libglib2.0-dev libudev-dev libmount-dev libdbus-1-dev libunwind-dev && \
+    apt-get clean
+RUN apt-get install -y libjpeg-dev libopenjp2-7-dev libgif-dev libtiff5-dev librsvg2-dev libheif-dev libwebp-dev libraw-dev \
+                       libpoppler-dev libpoppler-cpp-dev libspectre-dev libfreetype6-dev libfontconfig1-dev libharfbuzz-dev \
+                       libpulse-dev libsndfile1-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev \
+                       libibus-1.0-dev libscim-dev libfribidi-dev libinput-dev liblua5.2-dev libluajit-5.1-dev \
+                       libx11-dev libxext-dev libxrender-dev libxcursor-dev libxcomposite-dev libxinerama-dev libxrandr-dev \
+                       libxtst-dev libxss-dev libxdamage-dev libgl1-mesa-dev xvfb && \
```

Comment:
Wow, now we can run test shell in docker?

Reply:
Actually, this docker image is for building the `engine` repository and `*-unitests` can be ran after building the engine on this docker. 
#### for example:
```sh
# docker run -it ghcr.io/flutter-tizen/build-engine /bin/bash
mkdir /workspace && cd /workspace
git clone --depth 1 https://github.com/flutter-tizen/engine src/flutter
gclient-prepare-sync.sh --reduce-deps --shallow-sync
gclient sync --no-history
build-engine.sh 
./src/out/host_debug/flutter-tizen-unittests 
```

But if the `test shell` you mentioned refers to the environment for testing the flutter widgets, we will need another docker environment.